source/Calamari/Kubernetes/Scripts/KubectlBashContext.sh

```diff
@@ -229,7 +229,9 @@ function setup_context {
 function configure_kubectl_path {
   export KUBECONFIG=$(get_octopusvariable "Octopus.Action.Kubernetes.KubectlConfig")
   echo "Temporary kubectl config set to $KUBECONFIG"
-  echo "" >> $KUBECONFIG
+  # create an empty file, to suppress kubectl errors about the file missing
+  echo "" > $KUBECONFIG
+  chmod 600 $KUBECONFIG
```

Comment:
small _(personal)_ thing _(not necessarily need to be changed): I like to use the `chmod o=rw <file>` format which _might_ make it easier to figure out what permissions we're adding (at least for me as I struggle to remember all the numbers in permissions 😁)